src/docs/concepts-transactions/default-negotiation-process/index.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Default negotiation process
+slug: default-negotiation-process
+updated: 2025-09-16
+category: concepts-transaction-process
+ingress:
+  Sharetribe has a default negotiation transaction process that enables
+  users to negotiate a price for the transaction. This article explains
+  how the negotiation process works.
+published: true
+---
+
+# Negotiation process overview
```

Comment:
I don't have that much prior knowledge of the technical details of this feature so these were the questions I had after reading the overview:

- How do I toggle between forward and reverse flows (is there a setting or are they different transaction process paths or are they both enabled by default)
- There is a mention that the process does not support bookings or stock. What kind of offering is the process optimised for
- I am having a hard time to picture the process flow. What do you think of there be a visualisation (a simplification) depicting the general flow?